PS4k/src/com/vitco/manager/action/types/SelectLogicAction.java

```diff
@@ -0,0 +1,49 @@
+package com.vitco.manager.action.types;
```

Comment:
The action could be used in the plugins maybe so I thought would be better outside.